docs/modules/user-guide/partials/ref_devfile-samples.adoc

```diff
@@ -3,5 +3,6 @@
 
 .Additional resources
 
-* link:https://github.com/devfile/api/issues[Creating devfiles].
-* link:https://github.com/devfile/docs[Documenting devfiles].
+To build and test devfiles, visit our link:https://github.com/devfile/api[GitHub API repository]. Here, you can also view open issues, so you can see the creation of devfiles.
```

Comment:
I'd remove `Here` and start the next sentence with `You`.  You might even be able to delete the also, I'm not sure yet. I think it depends on how the second thing I'm going to write goes.

>visit our link:https://github.com/devfile/api[GitHub API repository]. You can also view open issues...

When you go to the open issues, do you see other people creating devfiles? Who does the creation? I think it would be clearer if we had a who doing what kind of information here. Maybe, and I'm just riffing here so don't copy and paste, because I don't know the answer:
`You can also view open issues to see real examples of devfiles.` 
Or 
`You can also view open issues to see other users creating their own devfiles.`

Reply:
Is it "our" GitHub API repository? Or "your"?

Reply:
My opinion on "Here" and "also" is not as strong, but as Neal mentioned, I also think "see the creation" should be replaced by something more descriptive, like "see how users create them" or "find more information about the creation of devfiles".

Reply:
> Is it "our" GitHub API repository? Or "your"? If you mean "our", who is "us"? I'd specify this.

Good point. In general, avoid possessive pronouns if at all possible. In this case, just substitute "our" with "the".

To the second point (_"Here" and "also"_), just avoid fluff:

_Here, you can also view open issues, so you can see the creation of devfiles._

-> 

_To see the creation of devfiles, review the open issues._

Reply:
Thanks, y'all! Resolving as I have addressed these comments.